### PR TITLE
fix: #302

### PR DESCRIPTION
**Pull request by taoqf**
https://github.com/jemise111/react-native-swipe-list-view/issues/302
https://github.com/jemise111/react-native-swipe-list-view/pull/296

### CODE REVIEW

File: types/index.d.ts [hunk 1]
```diff
@@ -1,33 +1,27 @@
...
-}
-
-interface IPropsSwipeRow {
+interface IPropsSwipeRow<T> {
 	/**
 	 * Used by the SwipeListView to close rows on scroll events.
 	 * You shouldn't need to use this prop explicitly.
 	 */
-	setScrollEnabled?: (boolean) => void;
+	setScrollEnabled(enabel: boolean): void;
```
Comment (leonid-shevtsov) [inline types/index.d.ts]: `enabel` => `enable`

Comment (taoqf) [reply types/index.d.ts]: Thanks

**Comment by jemise111** [comment]
@taoqf this is awesome, thanks so much!

Going to merge this but out of curiosity what is the purpose of the `types/test.tsx` file?

**Comment by taoqf** [comment]
> @taoqf this is awesome, thanks so much!
> 
> Going to merge this but out of curiosity what is the purpose of the `types/test.tsx` file?

Just make sure the definition is right. make no sense when is published.

**Pull request merged**